std/internal/cstring.d

```diff
@@ -111,26 +111,25 @@ if (isSomeChar!To && (isInputRange!From || isSomeString!From) &&
         auto r = cast(const(CF)[])str;  // because inout(CF) causes problems with byUTF
         if (r is null)  // Bugzilla 14980
         {
+            res._length = 0;
```

Comment:
This is a fix; _length was previously undefined due to void-initialization.

Reply:
Are there even cases where `res` won't be `null` from `trustedVoidInit`?
See also: https://issues.dlang.org/show_bug.cgi?id=14980

But we can try this in another PR ;-)

Reply:
`res` is the void-initialized struct, so all of its members are undefined. Just setting the ptr to null but leaving the length undefined doesn't sound too good.